website/.env

```diff
@@ -0,0 +1,3 @@
+ALGOLIA_APP_ID=YY0FFNI7MF
+ALGOLIA_INDEX=poc_NOMAD
+ALGOLIA_SEARCH_ONLY_API_KEY=5037da4824714676226913c65e961ca0
```

Comment:
@pruett you're going to need to remove this file and the burn this API key please.

Reply:
These values are ok to be public. They're shipped in frontend code.

Reply:
Hm, interesting. It feels like these should be consts in website/scripts/index_search_content.js then, rather than floating around as free files like this.